commit 7f3f4f7f6ad3
Author: Francois Marier <francois@mozilla.com>
Date:   Fri Nov 3 11:44:43 2017 -0700

    Bug 1393627 - Expose the full tracking protection UI preferences. r=mossop, a=ritu
    
    MozReview-Commit-ID: KyDBuHyDPLP
    
    --HG--
    extra : source : 516690579442aecc1bea191e905f019acfad5b48
---
 browser/app/profile/firefox.js | 4 ----
 1 file changed, 4 deletions(-)

diff --git browser/app/profile/firefox.js browser/app/profile/firefox.js
index 137462dc857e..9e08a0f166f9 100644
--- browser/app/profile/firefox.js
+++ browser/app/profile/firefox.js
@@ -1511,11 +1511,7 @@ pref("browser.ping-centre.production.endpoint", "https://tiles.services.mozilla.
 // Enable GMP support in the addon manager.
 pref("media.gmp-provider.enabled", true);
 
-#ifdef NIGHTLY_BUILD
 pref("privacy.trackingprotection.ui.enabled", true);
-#else
-pref("privacy.trackingprotection.ui.enabled", false);
-#endif
 pref("privacy.trackingprotection.introCount", 0);
 pref("privacy.trackingprotection.introURL", "https://www.mozilla.org/%LOCALE%/firefox/%VERSION%/tracking-protection/start/");
 
